ui/web/tpl/DashboardView.html

```diff
@@ -15,14 +15,13 @@ <h3 class="page-header">
+                            <img src="img/machine-diagram.jpg"
```

Comment:
i don't think we need to keep this diagram in the dashboard view anymore. it should be replaced with hardware(sensor) communication status(online/offline). remember this?